lib/phoenix/config.ex

```diff
@@ -72,11 +72,16 @@ defmodule Phoenix.Config do
   def from_env(otp_app, module, defaults) do
     otp_app
     |> fetch_config(module)
-    |> merge(defaults)
+    |> override(defaults)
   end
 
-  @doc false
-  def merge(a, b), do: Keyword.merge(b, a, &merger/3)
+  @doc """
+  Take 2 keyword lists and override values from first one with values from
+  the second one recursively.
```

Comment:
You are describing merge here, and not overrides. I think this is a strong sign we should just go with merge please? Less chance of confusion. :)